app/case-studies/[slug]/page.tsx

```diff
@@ -0,0 +1,62 @@
+import CaseDetailsDescription from "@/components/CaseDetailsDescription";
+import CaseDetailsFigma from "@/components/CaseDetailsFigma";
+import CaseDetailsHeader from "@/components/CaseDetailsHeader";
+import CaseDetailsLearnings from "@/components/CaseDetailsLearnings";
+import CaseDetailsOtherStudies from "@/components/CaseDetailsOtherStudies";
+import CaseDetailsProblemStatement from "@/components/CaseDetailsProblemStatement";
+import CaseDetailsProcess from "@/components/CaseDetailsProcess";
+import CaseDetailsTechStack from "@/components/CaseDetailsTechStack";
+import ContactBanner from "@/components/ContactBanner";
+import { client, urlFor } from "@/lib/sanity";
+import { Project } from "@/types";
+
+// export const revalidate = 10;
```

Comment:
If it works without can you remove it?

Reply:
@Nassim-Saboundji I commented it out in case when I restart the server it starts happening again. I will get rid of it once I know I don't need it!